File: InterfazJava/ventanasProyecto/src/clases/Producto.java
package clases;
import java.util.*;
import java.util.ArrayList;
import java.util.List;
import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.text.ParseException;
import java.util.Date;
public class Producto{
    private int idProducto;
    private String nombre;
    private double precio;
    private int cantMinima;
    private String marca;
    private String moneda;
    private String descripcion;
    private int esConsumible;

    @Override
    public String toString() {
        return nombre ;
    }

    public String getDescripcion(){
	return descripcion;
    }

    public void setDescripcion (String _descripcion){
        this.descripcion=_descripcion;
    }
    
    public String getMoneda(){
        return moneda;
    }
    
    public void setMoneda(String moneda){
        this.moneda = moneda;
    }
    
    public int getIdProducto(){
    	return idProducto;
    }

    public void setIdProducto (int _idProducto){
	this.idProducto=_idProducto;
    }

    public String getNombre(){
    	return nombre;
    }

    public void setNombre (String _nombre){
	this.nombre=_nombre;
    }

    public double getPrecio(){
	return precio;
    }

    public void setPrecio (double _precio){
	this.precio=_precio;
    }

    public int getCantMinima(){
	return cantMinima;
    }

    public void setCantMinima (int _cantMinima){
        this.cantMinima=_cantMinima;
    }

    public String getMarca(){
	return marca;
    }

    public void setMarca (String _marca){
	this.marca=_marca;
    }
    public int getEsConsumible(){
	return cantMinima;
    }

    public void setEsConsumible (int _esConsumible){
        this.cantMinima=_esConsumible;
    }

    public Producto(int idProducto,String nombre,double precio,int cantMinima,String marca){
	this.idProducto=idProducto;
	this.nombre=nombre;
	this.precio=precio;
	this.cantMinima=cantMinima;
	this.marca=marca;
	
    }
    public Producto(){
        
    }
}

File: InterfazC#/20132952_v4/Modelo/Producto.cs
using System;
using System.Collections.Generic;
using System.Text;


public class Producto{

    private int idProducto;
    private string nombre;
    private double precio;
    private string marca;
    private string descripcion;
    private int cantMinima;
    private int consumible;

    public Producto(int idProducto, string nombre, float precio, string marca, string descripcion)
    {
        IdProducto = idProducto;
        Nombre = nombre;
        Precio = precio;
        Marca = marca;
        Descripcion = descripcion;
    }

    public Producto()
    {
    }

    public int IdProducto{
        set{
            this.idProducto = value;
        }
        get{
            return idProducto;
        }
    }

    public string Nombre{
        set{
            this.nombre = value;
        }
        get{
            return nombre;
        }
    }

    public double Precio
    {
        set{
            this.precio = value;
        }
        get{
            return precio;
        }
    }

    public string Marca{
        set{
            this.marca = value;
        }
        get{
            return marca;
        }
    }

    public string Descripcion{
        set{
            this.descripcion = value;
        }
        get{
            return descripcion;
        }
    }

    public int CantidadMin
    {
        set
        {
            this.cantMinima = value;
        }
        get
        {
            return cantMinima;
        }
    }

    public int Consumible { get => consumible; set => consumible = value; }





    /*public int IdProducto { get => idProducto; set => idProducto = value; }
    public string Nombre { get => nombre; set => nombre = value; }
    public float Precio { get => precio; set => precio = value; }
    public string Marca { get => marca; set => marca = value; }
    public string Descripcion { get => descripcion; set => descripcion = value; }
    public DateTime Vencimiento { get => vencimiento; set => vencimiento = value; }*/
}

